### PR TITLE
Update Ember-can

### PR DESCRIPTION
**Pull request by Matthijsy**
### Summary
Time for a little bigger update, Ember-can. The CanMixin is removed from the source project so we need to use the service. For the routes I've injected this globally (since we use it in almost every route) and for the other entities I've injected them manually.

Also the abilities need to be in a singular form now, so I've renamed all the files and merged the cases when they existed both. I did not rename all the occurances, since this is done automatically by ember-can but if we like to do this there as well we could do that.

### CODE REVIEW

File: app/initializers/service-inject.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export default {
+  name: 'service-inject',
+
+  initialize(application) {
+    application.inject('route', 'can', 'service:can');
```
Comment (cmitz) [inline app/initializers/service-inject.js]: Do we also need this pattern for `session`?

Comment (Matthijsy) [reply app/initializers/service-inject.js]: In the routes we rarely use the session, but we can use it where we want. What do you suggest?

Comment (cmitz) [reply app/initializers/service-inject.js]: Ah, there is my answer. If we barely use it we don't need it here.

We'll see if the need arises in the future when mixins are gone

Comment (Matthijsy) [reply app/initializers/service-inject.js]: We don't need this file actually, but then we need to say `can: service()` if every file we use it (and I don't like that for the routes). So this is actually an auto inject for all routes.

Comment (cmitz) [reply app/initializers/service-inject.js]: I know :)

**Pull request merged**